### ISSUE TITLE
[OS X] Ctrl+F2 keyboard shortcut to focus menu bar does not work

### ISSUE THREAD

**Issue by miniak** [electron/electron#6016]
* Electron version: 1.2.2
* Operating system: OS X 10.9.5

Pressing Ctrl+F2 should focus the menu bar. It works in all native Mac applications. It does nothing in Electron apps. This is a quite serious accessibility issue.
